lib/game.rb

```diff
@@ -38,10 +46,26 @@ def switch_players
   def ask_move(player)
```

Comment:
Maybe this method would be better associated with the Player class than here in the game class.